dev/reference/sql/statements/split-region.md

```diff
@@ -0,0 +1,161 @@
+# Split region 使用文档
```

Comment:
Please add the metadata at the beginning of the file, and add this file to TOC.md.